### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by majora2007** [Kareadita/Kavita#352]
After performing a scan with NVDA, there are a multitude of accessibility issues.

**Comment by magujun** [comment]
I guess this will never be closed.
As new features come in, also comes the need to check for accessibility.
Is there any way to make it not an issue but a mandatory implementation check/step?

**Comment by majora2007** [comment]
What you're talking about are Non-Functional Requirements. Accessibility is one NFR of this project. This issue isn't closed because I never got around to doing the individual items. It's a low priority item at this point since so much has changed. What should be done is to do an accessibility audit in an upcoming release for the project and fix things that way.
